doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,72 @@
+Traffic Shaping
+===============
+
+Linux offers a very rich set of tools for managing and manipulating the transmission of packets. And
+since the host system is Linux based, those tools can be used to manage and shape traffics on Software
```

Comment:
We don't need to mention that this is a Linux based software, that is sort of implicit.